finesse/sensor_reading.py

```diff
@@ -5,16 +5,16 @@
 
 
 @dataclass
-class EM27Property:
+class SensorReading:
     """Class for representing EM27 monitored properties.
 
     Args:
-        name: name of the physical quantity
+        description: name of the physical quantity
```

Comment:
I think this might (though only might) be slightly (and only slightly) confusing as the table from which the EM27 properties are read contains a "Name" field and a "Description" field.

Reply:
Hmm... I don't mind whether we go with `name` or `description`. The reason I went with `description` is because elsewhere I've used that to mean a *human-readable* name (though seemingly I forgot to mention this in the docstring :upside_down_face:)

Reply:
I reverted to calling it `name`, but added a comment saying it should be human readable